engine/src/main/java/org/terasology/rendering/dag/nodes/AmbientOcclusionPassesNode.java

```diff
@@ -75,9 +79,9 @@ public void process() {
         if (renderingConfig.isSsao()) {
             PerformanceMonitor.startActivity("rendering/ambientOcclusionPasses");
             // TODO: consider moving these into initialise without breaking existing implementation
-            sceneOpaque = frameBuffersManager.getFBO("sceneOpaque");
-            ssaoBlurredFBO = frameBuffersManager.getFBO("ssaoBlurred");
-            ssaoFBO = frameBuffersManager.getFBO("ssao");
+            sceneOpaque = dynamicFBM.getFBO(DefaultDynamicFBOs.ReadOnlyGBuffer.getResourceUrn());
```

Comment:
Also: 
- I'd remove DefaultDynamicFBOs from the line (static import?)
- I'd rename getResourceUrn() to getName()
- I imagine the getFBO() call increases usage count for that FBO. Is it possible for a node to release the FBO more than once, by mistake?